lms/djangoapps/gradebook/management/commands/tests/test_generate_gradebook_entries.py

```diff
@@ -29,7 +33,7 @@ class GenerateGradebookEntriesTests(TestCase):
     def setUp(self):
 
         # Turn off the signalling mechanism temporarily
-        settings._wrapped.default_settings.FEATURES['SIGNAL_ON_SCORE_CHANGED'] = False
+        #settings._wrapped.default_settings.FEATURES['SIGNAL_ON_SCORE_CHANGED'] = False
```

Comment:
This line has been commented out -- is it intended to stay off?  Should it just be removed?